semaphore/.semaphore/bin/setup-db

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env -S bash -e
+
+start-db
+
+rake db:setup
```

Comment:
`rails` is generally preferred but for some reason certain `rails` commands don't use `spring`.